Specs/Scene/PointAppearanceSpec.js

```diff
@@ -0,0 +1,139 @@
+/*global defineSuite*/
+defineSuite([
```

Comment:
Is this how the other geometry/appearance specs are organized?  I feel like we could combine the point geometry/appearance tests into one file.